### PR TITLE
Cifar10_parallel, AlexNet

### PR DESCRIPTION
**Pull request by msakarvadia**
Updated cifar10_parallel to run using cuDNN, updated AlexNet hyperparameter ranges

### CODE REVIEW

File: alexnet/problem.py [hunk 1]
```diff
@@ -5,54 +5,77 @@
 # sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 from torch_wrapper import use_knl  # noqa
 
-
+#NOTE: I have intentially left the ranges for some of the hyper-parameters large in order to facilitate a large number of combinations. There will be configurations of these ranges that cause errors related to kernels being larger than images, but I am choosing to ignore those cases for now because limiting the ranges would greatly reduce the valid combinations of hyper-parameters. Feel free to change ranges based on your needs.
```
Comment (felker) [inline alexnet/problem.py]: Can you wrap this (and any other long lines) at 88 characters?

https://github.com/felker/deephyper_pytorch_layers/blob/master/.pre-commit-config.yaml 

Should auto-format the code for you when you attempt to `git commit`, as long as `pip install pre-commit flake8 black`

File: cifar10_parallel/cifar10_parallel_run.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
+import time
```
Comment (felker) [inline cifar10_parallel/cifar10_parallel_run.py:1]: For future reference: it is better to `git mv cifar10_run.py cifar10_parallel_run.py` compared to `git rm` then `git add`

File: alexnet/problem.py (hunk 1 above)
Comment (felker) [reply alexnet/problem.py]: also I think it is nice to sign `# TODO(MS):` and `# NOTE(MS):` comments with the author's initials

Comment (msakarvadia) [reply alexnet/problem.py]: I have auto-formatted the code as per-instructions, however, I don't think comments are auto-formatted. I went ahead and manually formatted the comments to fit requirments.

Comment (felker) [reply alexnet/problem.py]: Yeah turns out comments and long strings wont automatically be fixed by `black`

**Pull request merged**